src/main/java/lt/akademija/itacademymanager/controller/JwtAuthenticationController.java

```diff
@@ -0,0 +1,45 @@
+package lt.akademija.itacademymanager.controller;
+
+import lombok.AllArgsConstructor;
+import lt.akademija.itacademymanager.config.JwtTokenUtil;
+import lt.akademija.itacademymanager.exception.user.UserNotFoundException;
+import lt.akademija.itacademymanager.payload.request.UserNewRequest;
+import lt.akademija.itacademymanager.payload.response.JwtResponse;
+import lt.akademija.itacademymanager.service.UserService;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.authentication.AuthenticationManager;
+import org.springframework.security.authentication.BadCredentialsException;
+import org.springframework.security.authentication.UsernamePasswordAuthenticationToken;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@AllArgsConstructor
+public class JwtAuthenticationController {
+
+    private final AuthenticationManager authenticationManager;
+    private final JwtTokenUtil jwtTokenUtil;
+    private final UserService userService;
+
+    @PostMapping(value = "api/login")
+    public ResponseEntity<JwtResponse> createAuthenticationToken(@RequestBody UserNewRequest authenticationRequest) {
```

Comment:
prisijungimui naudočiau kitą klasę vietoje UserNewRequest

Reply:
+1

Reply:
Be to pattern UserNewRequest I am not a fan of it. Butu geriau pvz CreateUserRequest. New ir Old zodziai tokie time bound jeigu juos galima taip pavadint. Kiek laiko new bus new :D